packages/atomic/src/components/atomic-refine-toggle/atomic-refine-toggle.tsx

```diff
@@ -36,7 +37,7 @@ export class AtomicRefineToggle implements InitializableComponent {
   public initialize() {
     this.searchStatus = buildSearchStatus(this.bindings.engine);
     this.modalRef = document.createElement('atomic-refine-modal');
-    this.bindings.interfaceElement.prepend(this.modalRef);
+    this.host.insertAdjacentElement('beforebegin', this.modalRef);
```

Comment:
One of the issues that caused `updateBreakpoints` not to work was that it looks for an ancestor search layout, but the modal was appended to the interface directly, meaning the modal could never be inside the search layout.